base-math/truncdddf.c

```diff
@@ -49,7 +52,20 @@ CONVERT_WRAPPER(
 	  {
 	    if (DFP_EXCEPTIONS_ENABLED)
 	      DFP_HANDLE_EXCEPTIONS (FE_OVERFLOW|FE_INEXACT);
-	    return SIGNBIT(a) ? -INFINITY : INFINITY;
+
+		/* Default result when trap is disable - IEEE754 */
```

Comment:
I did not understand this comment...  Did you mean something in the lines of:

Correctly round the result according to the current rounding direction mode.

Reply:
@gftg85  That weird comment I forgot to remove. That's was for remind me that POWER ISA 2.07 says (pag. 125) we should adjust the exponent of the intermediate result and adjust the round result if the overflow exception is enable. If it's disable we should do as I implement. It's not clear to me if in this case "obvious overflow" when cast to double if we need to adjust the exponent and return as "Normal Number" or just do what I am doing.

Reply:
Section 4.4.3 of POWER ISA 2.07 only refers to operations with binary floating-point formats.  Section 5.5.10.4 refers to operations with decimal floating-point formats.  None of them describe how conversions between these floating-point types work. Moreover, they are specific to POWER.

On the other hand, ISO-IEC TR 24732, in section 6.2, describes how a C implementation of decimal floating-point type should make the conversions between floating-point types.  Based on that, I think that your implementation is correct.